app/assets/v2/js/animate.min.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
instead of add many `eslint-disable` comment in files, you can list all of them in `.eslintignore` instead so it will easier to track and fix these lint issues

Reply:
Yea I can actually do that as well.  Was only just adding it to each file since I was going through them anyway and that way I can remind myself to remove them when I get to the file and not have to edit the `eslint` file each time.

Thoughts?

Reply:
If you plan to fix all lint by yourself at end, you can pick whatever way you like. The plan is always changing though. List all of them in single file will help others have a clear picture of which files need to pickup and fix

Reply:
I was planning on it but only for purpose of "feature" commit.  

Since you put it like that though, which I didn't think about, you are absolutely right!

I'll consolidate that into 1 file for sake of others contributing to that specific case.

Good call @gasolin